src/mixins/itext_behavior.mixin.js

```diff
@@ -25,8 +25,9 @@
      */
     initAddedHandler: function() {
       var _this = this;
-      this.on('added', function() {
-        var canvas = _this.canvas;
+      this.on('added', function (opt) {
+        //  make sure we listen to the canvas added event
```

Comment:
is this a comment of somthing to fix or is this comment explains what the line below is doing? ( same for the below instance )

Reply:
explanation

Reply:
well so i don't get it.
We are on the added event handler so we are listenting to it.
And we are putting the canvas in a variable from the add event rather than the _this context.
Why does it get that comment?

Reply:
Because now that group fires `added|removed` these callbacks got triggered once iText entered/exited group which is undesriable

Reply:
maybe then added on canvas and added on group shouldn't be the same event. They are pretty much different.
Why did we add the added to group event?
A user of an app does not have a way to add things in a group without the developer writing some code . Why instead of using the event the developer can't add the desired code in his function?

Usually we add events only for things that are out of the developer control